src/tree-internal.component.ts

```diff
@@ -108,6 +116,18 @@ export class TreeInternalComponent implements OnInit {
     this.treeService.fireNodeMoved(addedSibling, e.captured.tree.parent);
   }
 
+  public getTreeAPI(): TreeInternalAPI {
+    return this.api;
+  }
+
+  public getChildAPIById(id: number | string): TreeInternalAPI {
```

Comment:
it is not quite child's API... Cause API for any node with an id can be acquired here.